pkg/controller/core/workload_controller.go

```diff
@@ -660,9 +661,12 @@ func workloadStatus(w *kueue.Workload) string {
 	if apimeta.IsStatusConditionTrue(w.Status.Conditions, kueue.WorkloadFinished) {
 		return finished
 	}
-	if workload.HasQuotaReservation(w) {
+	if workload.IsAdmitted(w) {
```

Comment:
do we really need this distinction? Can we just track `quotaReserved`?

Reply:
Yeah, I was thinking about it, and chose this approach because:
1. just renaming to `quotaReserved` would mean we still have misleading logs, but the other way round
2. having more granular status might be helpful, for example [here](https://github.com/kubernetes-sigs/kueue/blob/60b4674c02f7e4be36676688c30f7bd1edde1b43/pkg/controller/core/workload_controller.go#L580) we only need to enter for admitted

I guess if we have too many occurences of `(quotaReserved || admitted)` then we could wrap them with `quotaReservedOrAdmitted` helper function.